pyani_plus/public_cli.py

```diff
@@ -990,8 +992,11 @@ def cli_classify(  # noqa: PLR0913
 
     method = run.configuration.method
 
-    identity = run.identities
-    if identity is None:
+    if mode == "identity":
+        matrix = run.identities
+    elif mode == "tANI":
+        matrix = run.tani
+    if matrix is None:
```

Comment:
Start with a ``matrix=None`` line, or do that in an else clause, to solve the Codacy warning about using matrix before assignment.